here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/PluginCache.java

```diff
@@ -175,19 +179,24 @@ static class StorageConstructorByClassNameMap extends ConcurrentHashMap<String,
    */
   static <CONFIG> @Nullable Fe1<IStorage, CONFIG> wrapStorageConstructor(
       @NotNull Constructor<? extends IStorage> constructor, @NotNull Class<CONFIG> configClass) {
+    log.info("Wrapping constructor: {}", constructor);
```

Comment:
should we convert logs in the method as DEBUG, as tests are stable now? (in cloud we will use INFO logging)